node/basic-cloud-functions-bot/README.md

```diff
@@ -12,4 +13,5 @@ messages from a room. The sample is built using JavaScript on Google Cloud Funct
      Make sure to register the URL for the App Engine instance as the
      **HTTP endpoint** of the bot.
   1. Add the bot to a room or direct message.
-  2. Send the message to the bot with an @-message or directly in a DM, example: `@BotName i love dogs`
+  2. Send the message to the bot with an @-message or directly in a DM, example:
+     `@BotName i love dogs`
```

Comment:
Capitalize "i"?
`@BotName I love dogs.`

Reply:
Not looking to make rewrites to the logic, merely reorganize and lint. Not fixed.